### ISSUE TITLE
Focus indicator on blue buttons not accessible

### ISSUE THREAD

**Issue by zqian** [tl-its-umich-edu/canvas-app-explorer#655]
Based on the [Accessibility Review Report](https://docs.google.com/spreadsheets/d/1vt4uJuSq3itNu8Ouq6KI-4KOdOlOi_T9PoqOVy0e3T0):

Severity: 3 - Major

Test: All pages

Issue notes: 
Focus indicator on blue buttons is not accessible (Help, Scan, Rescan, Begin Review, Go Back buttons): 
 - The color change is too low contrast (should be at least 3:1)
 - The focus state is indicated only with color (better to have another way to see it, such as outlining the button)
 - The moving focus indicator can be disorienting (should not have motion)

Example of focused button: (see the report)

Category: Keyboard and other Input

Criterion: 
[2.4.7 - Focus Visible](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1#focus-visible)
[2.4.13 - Focus Appearance](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)

Impact: 
People who use keyboard or assistive input device may be prevented from navigating content.

Most affects people who are/have: 
Low vision, Mobility disability, Neurodivergent
